source/math/FloatRotation3D.ooc

```diff
@@ -50,6 +50,7 @@ FloatRotation3D: cover {
 	init: func@ ~full(=_roll, =_pitch, =_yaw) { this _updateQuaternion() }
 	init: func@ ~default { this init(0.0f, 0.0f, 0.0f) }
 	init: func@ ~fromPoint(point: FloatPoint3D) { this init(point x, point y, point z) }
+	init: func@ ~fromQuaternion (=_quaternion)
```

Comment:
What happens if someone tries this:
```ooc
// q is a Quaternion from somewhere
t := FloatTransform3D new(q)
value := t x 
```
`_roll`, `_pitch_`, `_yaw` are not updated when the `_quaternion` is set.